### ISSUE TITLE
Use multiple panes

### ISSUE THREAD

**Issue by protist** [akrennmair/newsbeuter#461]
**Newsbeuter version (copy from `newsbeuter -v`)**: 
2.9
```
Compilation date/time: Dec  7 2015 04:47:17
System: Linux 4.8.13-1-ARCH (x86_64)
Compiler: g++ 5.3.0
ncurses: ncurses 6.0.20161224 (compiled with 6.0)
libcurl: libcurl/7.52.1 OpenSSL/1.0.2j zlib/1.2.8 libpsl/0.16.1 (+libicu/58.2) libssh2/1.8.0 (compiled with 7.46.0)
SQLite: 3.16.2 (compiled with 3.9.2)
libxml2: compiled with 2.9.2
```
Steps to reproduce the issue: Open Newsbeuter

**Other info you think is relevant**:

Many RSS readers use a three-pane window, with a pane of the left for the list of feeds, a pane in the middle for the list of articles, and another pane for the body of the article, either right on below the list of articles.

```
┌───────┬──────────┬─────────┐
│       │          │         │
│ List  │ List     │ Body    │
│ of    │ of       │ of      │
│ feeds │ articles │ article │
│       │          │         │
└───────┴──────────┴─────────┘

┌───────┬──────────┐
│       │ List of  │
│ List  │ articles │
│ of    ├──────────┤
│ feeds │ Body of  │
│       │ article  │
└───────┴──────────┘
```
The advantage of this is that there are two static "parental" lists, so it's less confusing when browsing feeds. It's easy to pull up the next unread article (or "mark all articles if this feed read", then next unread), while not losing your "place" in both lists.

This three-pane layout is offered in all other feed readers that I've previously used: RSSOwl, Akregator, and Tiny Tiny RSS. A similar layout is also present in mutt (with the sidebar patch), with three panes showing mailboxes, email list, and email body. With Newsbeuter specifically, my terminal is maximised, so there is plenty of room for this kind of layout.

**Comment by Minoru** [comment]
Hi,

>The advantage of this is that there are two static "parental" lists, so
>it's less confusing when browsing feeds. 
It's been a while since I used any three-pane layout, but I believe panes aren't static: marking an article read, for example, should update its entry in article list, and the feed list should be updated to show current number of unread items.

>It's easy to pull up the next
>unread article

Already possible with `n`.

>(or "mark all articles if this feed read", then next
>unread), while not losing your "place" in both lists.

Newsbeuter already has this covered via ^V and ^G: you can go back to parent dialog and open as many children as you want.

If none of this suits you, I'd accept a patch adding the layout.

**Comment by protist** [comment]
Thanks for the quick answer.

>I believe panes aren't static

Ah yes, sorry. What I meant was that the position and general contents of each list was static. I agree that there should be slight changes as items are read. I meant to suggest that the fixed position was less confusing for me in keeping track of where I was up to.

> Already possible with… Newsbeuter already has this covered…

Yes, I certainly like those features, but I meant to suggest that these would work very well *with* a pane system.

> If none of this suits you, I'd accept a patch adding the layout.

Thank you for the offer, but my programming skills are not really in this area. I guess I was just feeling you out to see what your general thoughts were on this. I guess not much from the sounds of it. 😁 No problem at all! Also, depending on how you manage the project, I'm happy if you want to close this issue. Cheers!

**Comment by Minoru** [comment]
Yeah, I'm indeed not very enthusiastic about this, but I guess I'll keep the issue for now - I recall this same question being asked on IRC before.

Thanks for flying Newsbeuter!
